python/ray/rllib/agents/impala/vtrace_policy_graph.py

```diff
@@ -184,6 +185,15 @@ def to_batches(tensor):
             clip_rho_threshold=self.config["vtrace_clip_rho_threshold"],
             clip_pg_rho_threshold=self.config["vtrace_clip_pg_rho_threshold"])
 
+        # Compute KL divergence metrics
+        model_dist = Categorical(self.model.outputs)
+        behaviour_dist = Categorical(behaviour_logits)
+        self.KLs = model_dist.kl(behaviour_dist)
+        self.mean_KL = tf.reduce_mean(self.KLs)
+        self.max_KL = tf.reduce_max(self.KLs)
+        self.min_KL = tf.reduce_min(self.KLs)
+        self.median_KL = tf.contrib.distributions.percentile(self.KLs, 50.0)
```

Comment:
How about we drop min and median as well? Min is probably not very interesting and median is ~= mean.